assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -41,10 +41,6 @@ che.workspace.no_proxy=
 # Configures dns resolving servers. May contain several values separated by comma sign.
 che.dns.resolvers=
 
-# By default, when users access to a workspace with its URL the workspace
-# automatically starts if it is stopped. You can set this to false to disable this.
-che.workspace.auto_start=true
```

Comment:
Why have we removed that capability.
If removed it should have been deprecated !
cc @skabashnyuk

Reply:
Not clear what do you want? 
Make investigation why it was removed? Restore it in GWT ide?

Reply:
Ok, this property is going to stay, and I fixed it's missing functionality